### PR TITLE
[Android/TF-lite] method to set tf-lite delegate

### PR DESCRIPTION
**Pull request by jaeyun-jung**
parse custom option string to set tensorflow-lite delegate.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by again4you — APPROVED** [review]
LGTM!

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc [hunk 1]
```diff
@@ -923,6 +923,13 @@ tflite_parseCustomOption (const GstTensorFilterProperties *prop, tflite_option_s
 
         if (g_ascii_strcasecmp (pair[0], "NumThreads") == 0) {
           option->num_threads = (int)g_ascii_strtoll (pair[1], NULL, 10);
+        } else if (g_ascii_strcasecmp (pair[0], "Delegate") == 0) {
+          if (g_ascii_strcasecmp (pair[1], "NNAPI") == 0)
+            option->accelerators = "true:" ACCL_CPU_NEON_STR;
+          else if (g_ascii_strcasecmp (pair[1], "GPU") == 0)
+            option->accelerators = "true:" ACCL_GPU_STR;
+          else
+            ml_logw ("Unknown option to set tensorflow-lite delegate (%s).", strv[i]);
```
Comment (gichan-jang) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:932]: It is a duplicated expression. How about this?
```suggestion
            ml_logw ("Unknown option to set tensorflow-lite delegate (%s).", pair[1]);
```

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by dongju-chae — APPROVED** [review]
LGTM

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc (hunk 1 above)
Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc]: updated

**Review by myungjoo — APPROVED** [review]

**Pull request merged**